app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use Illuminate\Http\Request;
+use App\Post;
+use App\Models;
+use Illuminate\Support\Facades\Auth;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(){
+    
+        $posts =\App\Models\Post::all();
```

Comment:
Useを修正することにより、この様に短く書くことが出来るようになります
```suggestion
        $posts =Post::all();
```